.github/ISSUE_TEMPLATE/free-form.md

```diff
@@ -1,16 +1,2 @@
----
-name: Free form
-about: For everything that is NOT a bug report.
-title: ''
-labels: ''
-assignees: ''
-
----
-
-<!--
-
-DO NOT USE THIS TEMPLATE FOR BUG REPORTS!
-
-IF YOU DO, THEY WILL BE CLOSED AND MARKED AS INVALID!
-
--->
+## Ability to download older versions of Checkra1n?
```

Comment:
Hey

Reply:
Yoga

Reply:
Ghh y kg

Reply:
B Ku de

Reply:
Are you having a stroke? Plus I found a solution about this.

Reply:
I found where the older versions are. You guys can ignore this one and I do apologize.